_posts/0000-01-16-meetRanaFayez.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![yoda](https://cloud.githubusercontent.com/assets/16547949/25401233/2c9215d8-29c3-11e7-9e9b-d7cbccefb17a.jpg)
```

Comment:
I wonder if we can add a image desciption so people using screenreaders can "see" the picture